docs/modules/ROOT/pages/documentation/payara-server/request-tracing-service/tracing-remote-ejbs.adoc

```diff
@@ -0,0 +1,151 @@
+[[tracing-remote-ejbs]]
+= Tracing Remote EJBs
+
+`_Since Payara Server 5.2020.4_`
```

Comment:
```suggestion
_Since Payara Server 5.2020.4_
```

I'm sorry, I made a mistake in my suggestion.

Reply:
![image](https://user-images.githubusercontent.com/7734850/90032143-f5d83500-dcb5-11ea-96ad-0b7ae0af42aa.png)